server/routerlicious/packages/services-client/src/gitManager.ts

```diff
@@ -9,6 +9,7 @@ import { buildHierarchy } from "@fluidframework/protocol-base";
 import * as api from "@fluidframework/protocol-definitions";
 import { debug } from "./debug";
 import { ICreateRefParamsExternal, IPatchRefParamsExternal, IGitManager, IHistorian } from "./storage";
+import { IPragueGraphSummaryPayload, IWriteSummaryResponse } from "./storageContracts";
```

Comment:
Please don't use the old code name Prague

Reply:
Thanks for the heads up. I'll use something like Full or Complete to better convey intent